### PR TITLE
Governance RFC

### PR DESCRIPTION
**Pull request by dwillist**
Outline basic structures for `paketo-buildpacks` governance. [Readable](https://github.com/paketo-buildpacks/rfcs/blob/governance/accepted/0000-governance.md)

### CODE REVIEW

File: accepted/0000-governance.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+- Roles are Contributor, Maintainer, And Steering Commitee member, this limits what kinds of actions an individual may be allowed to make in the `paketo-buildpacks` org
+
+### Subteams
+Subteams are responsible for maintaining some set of repositories under the `paketo-buildpacks` org. A natural split is into "Language Teams" responsible for maintaining all buildpack & specification changes that pertain to a particular language ecosystem.
+
+In order to become a subteam member a user must be a contributor.
+
+Subteams have ownership & voting power over relevant `rfcs`.
+
+Both maintainers contributors & core-team members can be members on any number of Subteams.
```
Comment (mdelillo) [inline accepted/0000-governance.md:31]: Is this meant to include all three of: maintainers, contributors, and core-team members?

File: accepted/0000-governance.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+##### Additional Responsibilities:
+Each team is additionally responsible for maintaining their respective documentation at (paketo_website) as well as their README.md files in their respective repos.
+
+### Roles
+
+#### Contributor:
+Contributors are those who make regular contributions to the project (documentation, code reviews, responding to issues, participation in proposal discussions, contributing code, etc.). 
+
+New contributors may be self-nominated or be nominated by existing contributors, and must be elected by a supermajority of that project’s maintainers. Contributors may merge approved PRs for all subteams they are members of.
```
Comment (mdelillo) [inline accepted/0000-governance.md:41]: What percentage of the project's maintainers constitutes a supermajority?

File: accepted/0000-governance.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+Requirements for being nominated as a Contributor:
+
+- contribute 2 PR's under a single subteam.
+
+#### Maintainers
+Maintainers are in charge of the day to day maintenance of the team's projects. 
+
+They review, approve, and merge PRs in all subteams they are members of, ensuring contributions align with project goals and meet the project's quality standards.
+
+New maintainers must already be contributors, must be nominated by an existing maintainer, and must be elected by a supermajority of the steering commitee. Likewise, maintainers can be removed by a supermajority of the core team or can resign by notifying one of the maintainers.
```
Comment (mdelillo) [inline accepted/0000-governance.md:52]: What percentage of the steering committee or core team constitutes a supermajority?

File: accepted/0000-governance.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+The `paketo-community` github organization is designed as a staging ground for acceptance into the offically supported `paketo-buildpacks` org. Promotion from `paketo-community` into `paketo-buildpacks` requires an official [`rfc`](github.com/paketo-buildpacks/rfcs)
+
+## Contribution Model:
+
+The paketo contribution model is divided into the following, Subteams & Roles.
+
+- A subteam is a group responsible some set of repositories in the `paketo-buildpacks` org. This limits the repositories any individual is allowed to take action in.
+
+- Roles are Contributor, Maintainer, And Steering Commitee member, this limits what kinds of actions an individual may be allowed to make in the `paketo-buildpacks` org
```
Comment (mdelillo) [inline accepted/0000-governance.md:22]:
```diff
-Commitee
+Committee
```

File: accepted/0000-governance.md (hunk 3 above)
Comment (mdelillo) [inline accepted/0000-governance.md:52]:
```diff
-commitee
+committee
```

File: accepted/0000-governance.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+#### Maintainers
+Maintainers are in charge of the day to day maintenance of the team's projects. 
+
+They review, approve, and merge PRs in all subteams they are members of, ensuring contributions align with project goals and meet the project's quality standards.
+
+New maintainers must already be contributors, must be nominated by an existing maintainer, and must be elected by a supermajority of the steering commitee. Likewise, maintainers can be removed by a supermajority of the core team or can resign by notifying one of the maintainers.
+
+
+#### Steering Commitee
```
Comment (mdelillo) [inline accepted/0000-governance.md:55]:
```diff
-Commitee
+Committee
```

File: accepted/0000-governance.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+## Organizations:
+
+Paketo consists of two github organizations:
+
+- `paketo-buildpacks`
+- `paketo-community`
+
+The `paketo-buildpacks` github organization consists of actively supported buildpacks, libraries etc.
+
+The `paketo-community` github organization is designed as a staging ground for acceptance into the offically supported `paketo-buildpacks` org. Promotion from `paketo-community` into `paketo-buildpacks` requires an official [`rfc`](github.com/paketo-buildpacks/rfcs)
```
Comment (ForestEckhardt) [inline accepted/0000-governance.md:14]: Is paketo-community exclusively for staging things that before they go into paketo-buildpacks or are they just buildpacks that we are not including in paketo-buildpacks and therefore not in the official builder?

File: accepted/0000-governance.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
+# Paketo Governance RFC
+
+This RFC outlines the govornance structure for the Paketo project.
```
Comment (twoseat) [inline accepted/0000-governance.md:3]: -govornance
+governance

File: accepted/0000-governance.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+Each team is additionally responsible for maintaining  relevant documentation at paketo.io.
+
+### Roles
+
+#### Contributor:
+Contributors are those who make regular contributions to the project (documentation, code reviews, responding to issues, participation in proposal discussions, contributing code, etc.). 
+
+New contributors may be self-nominated or nominated by existing contributors, and must be elected by a supermajority of that project’s maintainers. Contributors may merge approved PRs, and create branches.
+
+Requirements for being nominated as a Contributor:
```
Comment (ForestEckhardt) [inline accepted/0000-governance.md:42]: I think that we should remove this and either let the subteams assign their own criteria or add to the later.

**Comment by nebhale** [comment]
@sclevine Can you please review the latest changes?

**Pull request merged**